src/cli/build/index.ts

```diff
@@ -43,4 +44,13 @@ export const build = async (args = process.argv.slice(2)) => {
       return;
     }
   }
+
+  const parsedCommandLine = readTsconfig(args, log);
+
+  if (!parsedCommandLine || process.exitCode) {
+    return;
+  }
+
+  const { options: compilerOptions } = parsedCommandLine;
+  await copyAssets(compilerOptions.outDir ?? 'lib');
```

Comment:
Should we skip this step if there is no `outDir` rather than defaulting to `lib`?